src/Aspire.Hosting.Azure.EventHubs/AzureEventHubsExtensions.cs

```diff
@@ -198,9 +225,27 @@ public static IResourceBuilder<AzureEventHubsResource> RunAsEmulator(this IResou
             // For the purposes of the health check we only need to know a hub name. If we don't have a hub
             // name we can't configure a valid producer client connection so we should throw. What good is
             // an event hub namespace without an event hub? :)
-            if (builder.Resource.Hubs is { Count: > 0 } && builder.Resource.Hubs[0] is { } hub)
+            if (builder.Resource.Hubs is [var hub])
```

Comment:
I think this will now throw if there is more than one hub registered on the resource? Is that desired behavior now?

Reply:
Why would it throw?

Reply:
`[var hub]` is a single-element match. If this fails, we'll fallback to this [else that throws](https://github.com/dotnet/aspire/blob/c5a98fa07c3cddc44600c50dba23559a0a4056f9/src/Aspire.Hosting.Azure.EventHubs/AzureEventHubsExtensions.cs#L251-L254).

Reply:
That isn't what we want. I'll test this.

Reply:
Good catch @captainsafia -- I'll revert the change.